setup.py

```diff
@@ -42,7 +42,6 @@
         'transaction',
         'Persistence',
         'zExceptions',
-        'ZODB3',
```

Comment:
Let's add `ZODB` and `BTrees` as an explicit dependency, since this product imports from them directly.